.golangci.yaml

```
@@ -139,7 +139,7 @@ linters-settings:
   nolintlint:
     # Exclude following linters from requiring an explanation.
     # Default: []
-    allow-no-explanation: [ funlen, gocognit, lll, exhaustruct, decorder, gomnd ]
+    allow-no-explanation: [ funlen, gocognit, lll, exhaustruct, decorder, mnd ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)